examples/todo/project/Dependencies.scala

```diff
@@ -0,0 +1,56 @@
+import sbt._
+
+object Dependencies {
+
+  object V {
+    val tessellation = "2.8.0"
+    val decline = "2.4.1"
+    val cats = "2.9.0"
+    val catsEffect = "3.4.2"
+    val organizeImports = "0.5.0"
+    val weaver = "0.8.1"
+    val pureconfig = "0.17.4"
+  }
+  def tessellation(artifact: String): ModuleID = "org.constellation" %% s"tessellation-$artifact" % V.tessellation
+
+  def decline(artifact: String = ""): ModuleID =
+    "com.monovore" %% {
+      if (artifact.isEmpty) "decline" else s"decline-$artifact"
+    } % V.decline
+
+  object Libraries {
+    val tessellationNodeShared = tessellation("node-shared")
+    val tessellationCurrencyL0 = tessellation("currency-l0")
+    val tessellationCurrencyL1 = tessellation("currency-l1")
```

Comment:
there should be a single dependency to tessellation sdk

Reply:
I can replace `node-shared` with `sdk` but am unable to use just `tessellation-sdk` as it does not include the dependencies for `currency-l1` and `currency-l0`

Reply:
Hmm, then we should add both to the `sdk`.
The `sdk` module should be the only one that external projects would depend on.